tests/c_api_test/test_.py

```diff
@@ -234,7 +234,7 @@ def test_booster():
             ctypes.byref(out_len),
             result.ctypes.data_as(ctypes.POINTER(ctypes.c_double)))
         if i % 10 == 0:
-            print('%d iteration test AUC %f' % (i, result[0]))
+            print(f'{int(i)} interation test AUC {result[0]:6f}')
```

Comment:
`int` is excess here.
Did you want to specify ``width`` or `precision` for AUC?
```
f'{value:{width}.{precision}f}'
```
```suggestion
            print(f'{i} interation test AUC {result[0]:.6f}')
```

Reply:
Thank you so much for the review, I missed that line number 32 required some changes, and for the second case I the value was printed as float that is why I typecasted into int but from your suggestion I understand where I am going wrong. 
Will submit a Pull request after updating these soon.

Reply:
Hello, I have made the changes that you have requested. Hope I haven't messed up. @StrikerRUS